crates/astria-sequencer/src/config.rs

```diff
@@ -14,6 +16,9 @@ pub struct Config {
     /// The path to the json encoded genesis file with a list of accounts.
     #[arg(long)]
     pub(crate) genesis_file: String,
+    /// The path to penumbra storage db.
+    #[arg(long)]
+    pub(crate) db_filepath: PathBuf,
```

Comment:
bikeshed (feel free to ignore): `db_filepath` is kinda excessive, `db_path` is probably fine.